doc/content/integrations/lr1110-geolocation/_index.md

```diff
@@ -0,0 +1,125 @@
+
+The [Device & Application Services with Geolocation for Modem-based Devices API](https://www.loracloud.com/documentation/device_management?url=v1.html) on LoRa Cloud™ allows interacting with Device & Application Services to make efficient use of the LR1110's geolocation features.
+
+## Prerequisites
+
+1. A user account on LoRa Cloud™.
+
+2. A Ubidots user account with an upgraded plan.
+
+{{< warning >}} The [UbiFunctions](https://help.ubidots.com/en/articles/2132086-analytics-ubifunctions-user-guide) module used in this guide is currently not available for the STEM plan. {{</ warning >}}
+
+{{< note >}} This guide assumes you have already added your LR1110-based tracking device to {{% tts %}}. See [Adding Devices](https://www.thethingsindustries.com/docs/devices/adding-devices/) section for a detailed info. {{</ note >}}
+
+## Prepare a LoRa Cloud™ Setup
+
+To prepare the setup for your device's location resolution, first go to [LoRa Cloud™](https://www.loracloud.com/portal) and select **LoRa Cloud Device & Application Services** after logging in to your account.
+
+{{< figure src="lora-cloud-das.png" alt="LoRa Cloud Device & Application Services" >}}
+
+On the left hand menu, navigate to the **Device Owners** tab and click the **Create new owner** button. Enter a **Name**, choose a **Plan** and select **Submit**.
+
+{{< figure src="device-owner.png" alt="Creating a device owner" >}}
+
+Next, navigate to the **Manage Devices** tab on the left hand menu and click the **Add devices** button. Enter your device's EUI and click the **Add devices** button.
+
+{{< figure src="add-device.png" alt="Adding a device" >}}
+
+If you look under the **Manage Tokens** tab, you can find or create a new token needed for the next step.
+
+{{< figure src="manage-tokens.png" alt="Manage tokens" >}}
+
+## Create a LoRa Cloud™ Integration
+
+The next step is to integrate {{% tts %}} with LoRa Cloud™.
+
+In {{% tts %}} Console, navigate to the **Integrations** menu and select **LoRa Cloud**.
+
+Paste the token from LoRa Cloud™ in the **Token** field and enable the **LR1110 demo encoding** by checking the box.
+
+{{< figure src="lora-cloud-integration.png" alt="LoRa Cloud integration" >}}
+
+At this point, your device will send the location updates to the LoRa Cloud™ Device & Application Services. After receiving your device's location solution from Device & Application Services, in your device's **Live data** tab in {{% tts %}}, you will find a message of the **Forward location solved message** type. This message contains the object of the following structure:
+
+```bash
+"location_solved": {
+      "service": "lora-cloud-device-management-v1-wifi",
+      "location": {
+        "latitude": 51.455274,
```

Comment:
Please cut two or three digits in the `latitude` and `longitude`. This is my _exact_ home address.

Reply:
I see the issue, so I cut 3 digits and zoomed out the map on the end too